app/common/src/Price.js

```diff
@@ -2,16 +2,16 @@
 
 import * as React from 'react';
 import { Text } from 'react-native';
+import type { StylePropType } from '@kiwicom/react-native-app-common';
```

Comment:
Why is there import from package, since `Price` is part of the `common` as well? :)

Reply:
It's wrong. I was actually thinking about using public interface even inside of this package (and I will probably do it at least in tests like React itself did), but this is a mistake. I will fix it...

Reply:
Fixed.